treeherder/etl/buildbot.py

```diff
@@ -893,6 +894,7 @@
 
     # All other unit tests, sorted alphabetically by symbol.
     "CPP Unit Tests": "Cpp",
+    "CPP and GTest Unit Tests": "Cpp|G",
```

Comment:
@chmanchester - I wonder if we could go with ``Cpp-G`` as the symbol instead?  I don't think we use the pipe anywhere else, but we do use a "-" in a couple places: ``R-e10s``, for example.